src/TOMLInterface.jl

```diff
@@ -301,6 +301,46 @@ function construct_2d_array(arr::Expr)
 end
 
 
+"""
+    save_parameter_samples(
+        distribution::ParameterDistribution{Samples},
+        default_param_data,
+        num_samples,
+        save_path;
+        save_file = "parameters.toml",
+        rng = Random.MersenneTwister(1234),
+    )
+Takes samples from the given `distribution` and saves them to individual TOML files
+in the folder specified by `save_path`
+
+Arguments:
+- `distribution` - ParameterDistribution{Samples} to sample from
+- `default_param_data` - Dict of default parameters to be combined and saved with
+the parameters in `param_array` into a toml file
+- `save_path` - Folder where the parameters will be saved
+- `save_file` - Name of the toml files to be generated
+- rng - The random number generator used in sampling
+"""
+function save_parameter_samples(
+    distribution::ParameterDistribution{Samples},
+    default_param_data,
+    num_samples,
+    save_path;
+    save_file = "parameters.toml",
+    rng = Random.MersenneTwister(1234),
+) where {Samples <: ParameterDistributions.Samples}
+
+    save_parameter_ensemble(
+        sample(rng, distribution, num_samples),
+        distribution,
+        default_param_data::Dict,
+        save_path,
+        save_file;
+        pad_zeros = 3,
```

Comment:
Is there a reason we hard-code this to 3?

Reply:
No, I set it to be an optional keyword argument.